app/models/maestrano/connector/rails/concerns/organization.rb

```diff
@@ -53,7 +53,17 @@ def initialize(attributes = nil, options = {})
 
     self.synchronized_entities = {}
     Maestrano::Connector::Rails::External.entities_list.each do |entity|
-      synchronized_entities[entity.to_sym] = {can_push_to_connec: !pull_disabled, can_push_to_external: !push_disabled}
+      begin
+        clazz = "Entities::#{entity.to_s.titleize.split.join}".constantize
```

Comment:
You can do `entity.to_s.titleize.tr(' ', '')`
Also a comment explaining the transformation would be welcome